modules/common/istio/istio-values.tpl

```diff
@@ -115,24 +115,24 @@ tracing:
         memory: 800Mi
       limits:
         cpu: 1
-        memory: 1.5Gi
+        memory: 3Gi
 
 prometheus:
   resources:
     requests:
       cpu: 200m
-      memory: 2Gi
+      memory: 1.5Gi
     limits:
-      cpu: 500m
-      memory: 4Gi
+      cpu: 800m
```

Comment:
If prometheus was being throttled during the test, we should be increasing the requests, not the limits.

Reply:
In my experience the request doesn't really effect throttling that much, in this case I was adjusting this prometheus pod's resource limits/requests to be equal to what it's set to in sandbox